docs/Calendar/BookingsDiagnosticSummary.md

```diff
@@ -0,0 +1,52 @@
+# BookingsDiagnosticSummary
+
+Download the latest release: [BookingsDiagnosticSummary.ps1](https://aka.ms/BookingsDiagnosticSummary)
+
+This script runs a series of tests in a bookings Mailbox (one per execution) and returns a summarized list of the bookings Mailbox characteristics, as well as testing for known configuration issues that can lead to bookings not performing as expected.
+
+Additionally, it will collect the most common logs needed for troubleshooting by support, including:
+
+* Staff Membership log
+* Message Tracing Log
+* Booking Mailbox configuration
+* Staff List and Permissions
+* Services configuration
+
+To run the script, you will need a valid SMTP Address for a booking Mailbox.
+
+The Identity parameter is required, all remaining are optional and default to true.
+
+| Parameters:                   | Explanation:                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
Look at SourceSideValidation.md file and try to use that same format for the docs.

Reply:
Added Syntax section as discussed